README.rst

```diff
@@ -1,6 +1,37 @@
 Cloud Spanner support for Django
 ================================
 
+`Cloud Spanner`_ is the world's first fully managed relational database service
+to offer both strong consistency and horizontal scalability for
+mission-critical online transaction processing (OLTP) applications. With Cloud
+Spanner you enjoy all the traditional benefits of a relational database; but
+unlike any other relational database service, Cloud Spanner scales horizontally
+to hundreds or thousands of servers to handle the biggest transactional
+workloads.
+
+
+- `Client Library Documentation`_
+- `Product Documentation`_
+
+.. _Cloud Spanner: https://cloud.google.com/spanner/
+.. _Client Library Documentation: https://googleapis.dev/python/django-google-spanner/2.2.1b1/index.html
```

Comment:
this i not how we should link the doc, otherwise it will always open 2.2.1b1 docs, we want to use the latest always. https://googleapis.dev/python/django-google-spanner/latest/index.html